deploy/docker/pump.io.json

```diff
@@ -0,0 +1,23 @@
+{
+    "driver":  "mongodb",
+    "params":  {"dbname": "pumpio", "host":"127.0.0.1", "port":27017},
```

Comment:
This is pointing to a MongoDB running on localhost, but the admin is supposed to run MongoDB outside containers based on this image. This won't work, AFAICT.